src/config/firstLaunch.js

```diff
@@ -0,0 +1 @@
+export const LAUNCHED = '/com.askscore.alreadyLaunched'; 
```

Comment:
Use a single config file. This method is not what I expected.

Reply:
Do you want me to create a Single file like Config.js for all the configuration purposes including firebase, firestore, and this launched variable too, or do I need to declare it as an environment variable?

Reply:
Since you are dealing with docker containers also, let's use a env file. Do not commit the original env file. Use gitignore to ignore that file. But commit a example env file containing exactly the same variable names. Something like we have done on go-social, TrackPal and Animal-Rescue.

Reply:
I will add environmental variables  in an .env file when I will setup Docker. Is that ok?

Reply:
Do not until that. Once you have initialize the project with firebase, please add the env file.